source/host/System.hh

```diff
@@ -10,10 +11,6 @@ public:
     void init();
     void run();
 
-    EGG::SceneManager *sceneMgr() const {
-        return m_sceneMgr;
-    }
-
```

Comment:
Why getting rid of this? Did you just notice that it's unused?

Reply:
Yes.